tools/build_node.js

```diff
@@ -173,8 +173,9 @@ async function buildNode(options) {
     if (stat.isDirectory()) return;
 
     if (file.endsWith(".css")) {
-      installCleanCSS(`./src/styles/${file}`, `styles/${file}`);
-      installCleanCSS(`./src/styles/${file}`, `scss/${file.replace(".css", ".scss")}`);
+      installCleanCSS(`./src/styles/${file}`, `styles/${file}`, { minify: false });
+      installCleanCSS(`./src/styles/${file}`, `styles/${file.replace(".css", ".min.css")}`, { minify: true });
+      installCleanCSS(`./src/styles/${file}`, `scss/${file.replace(".css", ".scss")}`, { minify: false });
```

Comment:
Doesn't this also need both minified and non-minified versions?

Reply:
Ah, good point.  I really don't know what these files are intended for (they predate me) - they aren't real SCSS (obviously)... they will be removed with v12 (as a breaking chance)... but until then this makes sense, I think we can un-minify them.